### PR TITLE
* Add API usage constraints section.

### PR DESCRIPTION
**Pull request by perusio**
Pull Request
============

### Scope of the PR

Add a section about the API usage constraints.

### Checklist

- [ x] Checked spelling and grammar
- [ x] Provided code samples where relevant
- [ x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — COMMENTED** [review]
I would just reformulate this: 
"Are there any limitations on using the UP42 API?"
"Yes. For more information, please consult our ..."

File: source/FAQ/api.rst [hunk 1]
```diff
@@ -30,11 +30,18 @@ Where can I learn more about the UP42 API?
 To learn more about our API and the concepts underlying it go to our
 :ref:`documentation <api-walkthrough>` or maybe you prefer to dive into the :ref:`API reference <api-specification>`.
 
+
+Are there any limitations on using the UP42 API?
```
Comment (bobthebuilderberlin) [inline source/FAQ/api.rst:34]: Not a native speaker, but would'nt it be `to` instead of `on`?

Comment (perusio) [reply source/FAQ/api.rst]: Sure.

**Comment by perusio** [comment]
> I would just reformulate this:
> "Are there any limitations on using the UP42 API?"
> "Yes. For more information, please consult our ..."

Done.

File: source/reference/api-usage-constraints.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+short period.
+
+Once exceeded that limit the API returns a client side  error using the `429
+Too Many Requests <https://httpstatuses.com/429>`_ HTTP status code.
+
+If you ever experience such an error code please do one of the
+following:
+
+ 1. Adjust the request rate from your UP42 API client application such
+    that the request rate follows an `exponetial backoff
```
Comment (bobthebuilderberlin) [inline source/reference/api-usage-constraints.rst:38]: * `exponential`

File: source/reference/api-usage-constraints.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <https://en.wikipedia.org/wiki/Exponential_backoff>`_ growth rate.
+ 2. Contact `support <mailto:support@up42.com>`_ to get that
+    restriction removed in case you have a very good reason for
+    that. If possible, this is only applicable to paying customers,
+    and **always**, in very special cases, and with a proper justification.
```
Comment (bobthebuilderberlin) [inline source/reference/api-usage-constraints.rst:43]: no need for the commas.

Comment (perusio) [reply source/reference/api-usage-constraints.rst:43]: That is a question of style. If you want to say everything in one sitting or if give some breathing room and continue later. So I remove the first two and leave the last.

Comment (perusio) [reply source/reference/api-usage-constraints.rst]: I ended rewriting the final part of the sentence. Conciseness is a big plus.

**Pull request merged**